packages/terra-modal/lib/Modal.scss

```diff
@@ -0,0 +1,34 @@
+@import '~terra-mixins';
+
+.terra-Modal {
+  background-color: #fff;
+  border-radius: 5px;
+  bottom: 10px;
+  display: block;
+  left: 10px;
+  max-height: 100%;
+  outline: none;
+  overflow: hidden;
+  position: fixed;
+  right: 10px;
+  top: 10px;
```

Comment:
Was a decision made about using px units vs rem? Will we just swing back through and use the new mixin for converting these when it's available? 
https://github.com/cerner/terra-core/issues/164